### PR TITLE
test(cli): reorganize e2e tests and add more tests for cli commands

### PR DESCRIPTION
**Pull request by ctrlc03**
# Description

Re organize tests:

- [x] remove redundant e2e tests
- [x] move keys tests from integration tests to cli folder
- [x] separate tests into folders

## Related issue(s)

re: #910 

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://github.com/privacy-scaling-explorations/maci/blob/dev/CONTRIBUTING.md) and [code of conduct](https://github.com/privacy-scaling-explorations/maci/blob/dev/CODE_OF_CONDUCT.md).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: cli/tests/unit/utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  });
+
+  describe("validateSalt", () => {
+    it("should return true for a valid salt", () => {
+      expect(validateSalt(genRandomSalt().toString())).to.eq(true);
+    });
+    it("should return false for an invalid salt", () => {
+      expect(validateSalt(SNARK_FIELD_SIZE.toString())).to.eq(false);
+    });
+  });
```
Comment (0xmad) [inline cli/tests/unit/utils.test.ts:33]:
```suggestion
  describe("vks", () => {
    it("should return true for two equal VKs", () => {
      expect(compareVks(vk1, vk1.asContractParam())).to.eq(true);
    });

    it("should return false for two unequal VKs", () => {
      const vk2 = vk1.asContractParam();
      vk2.alpha1.x = 9999n;
      expect(compareVks(vk1, vk2)).to.eq(false);
    });
  });

  describe("validateSalt", () => {
    it("should return true for a valid salt", () => {
      expect(validateSalt(genRandomSalt().toString())).to.eq(true);
    });

    it("should return false for an invalid salt", () => {
      expect(validateSalt(SNARK_FIELD_SIZE.toString())).to.eq(false);
    });
  });
```

File: tsconfig.json [hunk 2]
```diff
@@ -21,5 +21,5 @@
     "declaration": true
   },
   "exclude": ["node_modules"],
-  "include": ["./**/ts"]
+  "include": ["./**/ts", "cli/tests/unit/cli-genMaciKeypair.test.ts", "cli/tests/unit/genPubkey.test.ts"]
```
Comment (0xmad) [inline tsconfig.json:24]: Why do these tests included in bundle?

File: cli/tests/unit/data/testVk.json
Comment (0xmad) [inline cli/tests/unit/data/testVk.json:1]: Seems prettier wasn't run for this file.

**Review by 0xmad — CHANGES_REQUESTED** [review]
@ctrlc03 thanks, just few comments.

File: tsconfig.json (hunk 2 above)
Comment (ctrlc03) [reply tsconfig.json:24]: I have no idea 🤔 I dragged them over to cli/tests before changing name, maybe vs code added them for whatever reason? so odd haha will remove tho

File: cli/tests/unit/data/testVk.json
[the remark this answers is not in the document]
Comment (ctrlc03) [reply cli/tests/unit/data/testVk.json:1]: done

**Review by 0xmad — APPROVED** [review]
@ctrlc03 thanks!

**Pull request merged**